sdk/search/search-documents/src/searchIndexingBufferedSenderImpl.ts

```diff
@@ -75,9 +96,15 @@ class SearchIndexingBufferedSenderImpl<T> implements SearchIndexingBufferedSende
    */
   constructor(client: SearchClient<T>, options: SearchIndexingBufferedSenderOptions = {}) {
     this.client = client;
+    // General Configuration properties
     this.autoFlush = options.autoFlush ?? false;
-    this.flushWindowInMs = DEFAULT_FLUSH_WINDOW;
-    this.batchSize = DEFAULT_BATCH_SIZE;
+    this.batchDocumentCount = options.initialBatchDocumentCount ?? DEFAULT_BATCH_SIZE;
```

Comment:
why is this called `initialBatchDocumentCount` if it never seems to change? `initial` makes it sound like it's going to be modified over time

Reply:
Yes. That is the idea. There is a discussion that this might be opened up in future. So, Ted wanted to convey this message and wants to add the word 'initial'